browser-test/browser-test-compose.yml

```diff
@@ -2,6 +2,11 @@
 version: '3.4'
 
 services:
+  db:
+    volumes:
```

Comment:
**suggestion:** May be worth adding a comment that this should only be included for non-prod environments.

Reply:
done